### ISSUE TITLE
Review of 3.2 Package Metadata

### ISSUE THREAD

**Issue by rdeltour** [w3c/epub-specs#912]
> **accessMode** — the senses or faculties necessary to process or perceive the content (e.g., textual, visual, auditory, tactile).

I know this is coming from the schema.org definition, but "textual", "visual", etc are not "senses". Can be reworded to "identifies the senses or faculties..." or something else?

> **accessibilityFeature** — features and adaptations that increase the overall accessibility of the content (e.g., alternative text, extended descriptions, captions).

s/increase/participate to/ ? (some features do not really "increase" the accessibility, as some content is just not accessible without them).

> Inclusion of the following [schema.org] metadata is optional

s/metadata/accessibility metada/  (for consistency)

> accessibilityAPI — indicates the resource is compatible with the specified accessibility API (typically only used to identify [WAI-ARIA 1.1] conformance).

I still find that "compatible with the specified accessibility API" isn't properly defined. The clarification in the parenthesis helps, but can it be even stronger by removing the "typically"?

**Comment by mattgarrish** [comment]
I can set back to "a human sensory perceptual system or cognitive faculty". Visual is a perceptual system (meaning from images), and textual I assume falls under a cognitive faculty (meaning from words).

**Comment by mattgarrish** [comment]
I'd be cautious about trying to find wording that invalidates any usage we haven't considered in depth (flash?), or that might arise in the future.

We could try to clarify the definition more:

"indicates the publication is compatible with the specified accessibility API. This property is typically only used to indicate that the use of scripting in an EPUB Publication conforms to [WAI-ARIA 1.1], as compatibility with operating system accessibility APIs is a concern for Reading Systems."

Or something like that.

**Comment by mattgarrish** [comment]
I've made the changes above, plus switching "increases" to "contributes to". If you still think there's an issue feel free to reopen with alternatives.
